certbot/certbot/crypto_util.py

```diff
@@ -222,6 +223,32 @@ def verify_renewable_cert(renewable_cert):
     verify_cert_matches_priv_key(renewable_cert.cert_path, renewable_cert.key_path)
 
 
+def load_cert(cert_path):
+    """Reads the certificate PEM file and returns a cryptography.x509.Certificate object.
+
+    :param str cert_path: Path to the certificate
+
+    :rtype: `cryptography.x509.Certificate`
+    :returns: x509 certificate object
+    """
+    with open(cert_path, 'rb') as fh:
+        cert_pem = fh.read()
+    return x509.load_pem_x509_certificate(cert_pem, default_backend())
+
+
+def cert_sha1_fingerprint(cert_path):
```

Comment:
nit: Recently we fixed some issues where RHEL 7 FIPS systems were complaining about MD5. See https://github.com/certbot/certbot/pull/7708.

Those systems current do not complain about SHA-1, but I'm not sure the state of things in newer versions of FIPS and what it will be in the future. I tried testing it on CentOS 8 and had issues getting FIPS mode enabled.

Because of the potential for problems here, I'd like to move this function out of the public API and into the Apache plugin. If we end up having a use for it in other parts of Certbot, we can maybe move it here, but until then, I think keeping it private makes it easier for us to responsibly make any changes here such as confidently flagging that SHA-1 isn't being used for security purposes like we did with our MD5 code.